file-format-verification/README.md

```diff
@@ -0,0 +1,3 @@
+## File Format Verification Tool (work in progress)
+
+The File Format Verification Tool is an electronic file format verification tool for filers who wish to confirm that a LAR is formatted in the required pipe delimited text file format. This verification tool will be available on a webpage that runs independently from the HMDA Platform and provides a convenient test mechanism for filers. https://github.com/cfpb/hmda-platform
```

Comment:
Because this already runs in Docker, can we add the couple steps to get it running?